pkg/templates/images/images.go

```diff
@@ -299,7 +299,7 @@ func optionalResources() map[Resource]map[string]string {
 			"1.27.x":    "registry.k8s.io/provider-os/openstack-cloud-controller-manager:v1.27.3",
 			"1.28.x":    "registry.k8s.io/provider-os/openstack-cloud-controller-manager:v1.28.1",
 			"1.29.x":    "registry.k8s.io/provider-os/openstack-cloud-controller-manager:v1.29.0",
```

Comment:
Can we also update CCM and CSI for v1.28 and v1.29? There are newer versions, it might be a good idea to update those as well now that we're updating v1.30.